### DISCUSSION TITLE
How to improve the consistency of time between transmissions?

### DISCUSSION THREAD

**Discussion by DrCoconut1245** [mikaelnousiainen/RS41ng#70] [Q&A]
Hi, while testing the sonde I have noticed that packets come in at inconsistent intervals, anywhere from 1-5 seconds (using Hours V2). Does anyone know what is causing this, and if so how I could lower that gap? Would sending back less data per packet help (maybe its delay when waiting for some sensor data to get back?), and if so can that be changed? Is it just from noise, and I just am not decoding packets the sonde is sending back? Could I make Hours transmit more often than just once per second? I am very much a radio beginner, so any advice or suggestions are appreciated. All I really need to be transmitted is GPS time, lat, long, alt, and callsign. If that is possible in any way quickly on a RS41, I would love to know. Thank you!

**Comment by K4KDR** [comment]
Hello!  Best advice will come from the maintainers of this software, of course.

But just FYI, my preference was to transmit every -10- seconds, so I have the following line in **/src/config.h**:

```
// Delay after transmission for modes that do not use time synchronization. Zero delay allows continuous transmit mode for Horus V1 and V2.
#define RADIO_POST_TRANSMIT_DELAY_MS 10000
```

... where the beacons are broken up into -5- as 4FSK & -1- as APRS as defined in these lines:

```
// Which modes to transmit using the built-in Si4032 transmitter chip
// The COUNT settings define the number of times that each type of transmission is repeated
#define RADIO_SI4032_TX_CW false
#define RADIO_SI4032_TX_CW_COUNT 1
#define RADIO_SI4032_TX_PIP false
#define RADIO_SI4032_TX_PIP_COUNT 6
#define RADIO_SI4032_TX_APRS true
#define RADIO_SI4032_TX_APRS_COUNT 1
#define RADIO_SI4032_TX_HORUS_V1 false
#define RADIO_SI4032_TX_HORUS_V1_COUNT 1
#define RADIO_SI4032_TX_HORUS_V2 true
#define RADIO_SI4032_TX_HORUS_V2_COUNT 5
```

... I'm also a beginner w/ this hardware/software, but just wanted to share my setup.

Good luck!

-Scott,  K4KDR
